<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>760</width>
    <height>540</height>
   </rect>
  </property>
  <property name="palette">
   <palette>
    <active>
     <colorrole role="WindowText">
      <brush brushstyle="SolidPattern">
       <color alpha="217">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
    </active>
    <inactive>
     <colorrole role="WindowText">
      <brush brushstyle="SolidPattern">
       <color alpha="217">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
    </inactive>
    <disabled>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
    </disabled>
   </palette>
  </property>
  <property name="windowTitle">
   <string>Кошачий фотоальбом</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <spacer name="verticalSpacer_2">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>628</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="0" column="2">
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>628</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="1" column="0">
     <widget class="QPushButton" name="btn_left">
      <property name="styleSheet">
       <string notr="true">border:none</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="resources.qrc">
        <normaloff>:/buttons/images/chevron_l.png</normaloff>:/buttons/images/chevron_l.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>32</width>
        <height>32</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>733</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="1" column="2">
     <widget class="QPushButton" name="btn_right">
      <property name="styleSheet">
       <string notr="true">border:none</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="resources.qrc">
        <normaloff>:/buttons/images/chevron_r.png</normaloff>:/buttons/images/chevron_r.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>32</width>
        <height>32</height>
       </size>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>760</width>
     <height>37</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>меню</string>
    </property>
    <widget class="QMenu" name="menu_2">
     <property name="title">
      <string>Период переключения</string>
     </property>
     <addaction name="action_0sec"/>
     <addaction name="action_1sec"/>
     <addaction name="action_5sec"/>
     <addaction name="action_10sec"/>
    </widget>
    <addaction name="action_up_windows"/>
    <addaction name="action_use_resources"/>
    <addaction name="action_choose_dir"/>
    <addaction name="menu_2"/>
    <addaction name="action_close_app"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <action name="action">
   <property name="text">
    <string>Поверх всех окон</string>
   </property>
  </action>
  <action name="action_2">
   <property name="text">
    <string>Фото из файлов ресурса</string>
   </property>
  </action>
  <action name="action_3">
   <property name="text">
    <string>Фото из выбранной папки</string>
   </property>
  </action>
  <action name="action1">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Отключить</string>
   </property>
  </action>
  <action name="action5">
   <property name="text">
    <string>1 секунда</string>
   </property>
  </action>
  <action name="action5_2">
   <property name="text">
    <string>5 секунд</string>
   </property>
  </action>
  <action name="action10">
   <property name="text">
    <string>10 секунд</string>
   </property>
  </action>
  <action name="action_5">
   <property name="text">
    <string>Закрыть программу</string>
   </property>
  </action>
  <action name="action_up_windows">
   <property name="text">
    <string>Поверх всех окон</string>
   </property>
  </action>
  <action name="action_use_resources">
   <property name="text">
    <string>Фото из файла ресурсов</string>
   </property>
  </action>
  <action name="action_choose_dir">
   <property name="text">
    <string>Фото из выбранной папки</string>
   </property>
  </action>
  <action name="action_close_app">
   <property name="text">
    <string>Закрыть программу</string>
   </property>
  </action>
  <action name="action_0sec">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Отключить</string>
   </property>
  </action>
  <action name="action_1sec">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>1 секунда</string>
   </property>
  </action>
  <action name="action_5sec">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>5 секунд</string>
   </property>
  </action>
  <action name="action_10sec">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>10 секунд</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
